### PR TITLE
Update release.json and Go modules for 6/7.32.0-rc.2

### PR DESCRIPTION
**Pull request by gh123man**

### CODE REVIEW

**Review by remeh — APPROVED** [review]
LGTM as long as other tags (integrations-core etc.) did not change

**Review by olivielpeau — APPROVED** [review]
Since this blocks QA, let's not wait for a review from @DataDog/agent-platform to merge

**Pull request merged**
